integrationtest/dhtrestart/main.d

```diff
@@ -61,7 +62,7 @@ class RestartWithListeners : TestCase
 {
     import turtle.env.Dht;
 
-    import ocean.core.Test;
+    import Ocean = ocean.core.Test;
```

Comment:
Why is it necessary after renaming `test` package to `integrationtest`?

Reply:
Hm good question. I'll investigate.

Reply:
Ahh, that  commit comes _before_ the makd update.

Reply:
Reordered commits and removed the named import change.